Exception/Exception.php

```diff
@@ -1,28 +0,0 @@
-<?php
```

Comment:
I want to keep this class: if i need to throw an exception who is not a ConnectionException, i will use this one

Reply:
Maybe better to create an Interface Exception